doc/websocket.qbk

```diff
@@ -227,7 +227,7 @@ void echo(beast::websocket::stream<boost::asio::ip::tcp::socket>& ws)
 {
     beast::streambuf sb;
     beast::websocket::opcode::value op;
-    ws.read(sb);
+    ws.read(op, sb);
```

Comment:
Working my way through actually building something w/ Beast, so consuming all the existing docs in detail!